src/client/src/shell/components/app-header/Header.tsx

```diff
@@ -93,6 +93,9 @@ const IconButton = styled.button<{ type: string }>`
     color: ${({ theme }) => theme.button.secondary.textColor};
   }
 `
+IconButton.defaultProps = {
+  role: 'button',
+}
```

Comment:
According to MDN:

"Where possible, it is recommended to use native HTML buttons (`<button>`, `<input type="button" />`,  `<input type="submit" />`, `<input type="reset" />` and `<input type="image" />`) rather than the button role, as native HTML buttons are supported by all user agents and assistive technology and provide keyboard and focus requirements by default, without need for additional customization."

Reply:
You can use a markdown quote by preceding it with `>`

Reply:
Resolved IRL.